Makefile

```
@@ -14,7 +14,8 @@ all: test build
 
 build: builddir clean npm frontend sippy sippy-daemon
 
-verify: lint
+.PHONY: verify apm verify-apm
+verify: lint verify-apm
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)